pkg/cmd/attestation/inspect/inspect.go

```diff
@@ -21,70 +28,58 @@ import (
 func NewInspectCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command {
 	opts := &Options{}
 	inspectCmd := &cobra.Command{
-		Use:    "inspect [<file path> | oci://<OCI image URI>] --bundle <path-to-bundle>",
-		Args:   cmdutil.ExactArgs(1, "must specify file path or container image URI, as well --bundle"),
+		Use:    "inspect <path-to-sigstore-bundle>",
+		Args:   cmdutil.ExactArgs(1, "must specify bundle file path"),
 		Hidden: true,
-		Short:  "Inspect a sigstore bundle",
+		Short:  "Inspect a Sigstore bundle",
 		Long: heredoc.Docf(`
 			### NOTE: This feature is currently in public preview, and subject to change.
```

Comment:
Is this still in public preview?  Any idea on timeframe when this comes out to GA?

Reply:
This command TBH is more intended to be a debugging tool for our own team, so I'm inclined to keep it hidden outright / not be beholding to keeping the output in a particular way 🤔.